src/main/java/bit/javaoop/Company.java

```diff
@@ -0,0 +1,34 @@
+package bit.javaoop;
+
+public class Company {
+    private String name;
+    private CEO currentCEO;
+    private boolean doWeHaveCEO;
+
+    public Company(String name) {
+        this.name = name;
+        this.doWeHaveCEO = false;
+    }
+
+    public void hireCEO(CEO newCEO) {
+        this.doWeHaveCEO = true;
+        this.currentCEO = newCEO;
+    }
+
+    @Override
+    public String toString() { // to nie do konca zrozumialem czy o to chodzilo
```

Comment:
No prawie o to :D
Wszystko byłoby dobrze, gdybyś zamiast wypisywać na ekran, utworzył jednego długiego String'a, a na samym końcu go zwrócił, zamiast zwracać "Done".
Metoda publiczna toString() jest takim jakby słownym opisem obiektu. Ma nam zwrócić łańcuch znaków, który właśnie opisuje obiekt. I w naszym przypadku jest to struktura firmy. Czyli jeśli chcemy "zestringować" obiekt Company to dostajemy opis Company w postaci hierarchii firmy.